features/other_b/skin_wizard.feature

```diff
@@ -12,6 +12,32 @@ Feature: Skin wizard
   Then I should see "Create New Skin"
     And I should see "CSS"
 
+  @javascript
+  Scenario: User can add a parent skin to the wizard
+  Given I am logged in
+    And I create the skin "Dad"
+  When I go to the new skin page
+    And I follow "Use Wizard"
+  Then I should see "Site Skin Wizard"
+    And I should see "Parent Skins"
+  When I fill in "Title" with "Child"
+    And I follow "Add parent skin"
+    And I wait 1 second
+  Then I should see a parent skin text field
+
+  @javascript
```

Comment:
Since this isn't testing the wizard, it should probably be in features/other_b/skin.feature instead.